cmd/lakectl/cmd/doctor.go

```diff
@@ -127,7 +128,7 @@ func ListRepositoriesAndAnalyze(ctx context.Context) error {
 	case resp.JSON401 != nil:
 		return &CredentialsError{msgOnErrCredential, resp.JSON401.Message}
 	// In case we get the "not found" HTML page (the status is 200 and not 404 in this case).
-	case resp.HTTPResponse != nil && resp.HTTPResponse.StatusCode == 200:
+	case resp.HTTPResponse != nil && resp.HTTPResponse.StatusCode == 302:
```

Comment:
The comment doesn't change?

Reply:
Which comment? The output comment of the `doctor`?